docs/index.md

```diff
@@ -58,27 +58,27 @@ Community <https://docs.flyte.org/en/latest/community/index.html>
 :hidden:
 
 Introduction to Flyte <self>
-getting_started/quickstart_guide
-getting_started/getting_started_with_workflow_development
-getting_started/flyte_fundamentals
-getting_started/core_use_cases
+Quickstart guide <https://docs.flyte.org/en/latest/quickstart_guide.html>
```

Comment:
An example of this behavior is the `ChatGPT agent` link in https://docs.flyte.org/en/latest/deployment/agents/chatgpt.html#deployment-agent-setup-chatgpt
<img width="850" alt="image" src="https://github.com/flyteorg/flytesnacks/assets/2816689/7de6f1fb-865a-4399-ba36-1cfded055221">

Reply:
I think it's okay if there are external links in the index file in this repo since the actual index file for the docs site lives in the flyte repo. Anyone who builds flytesnacks docs locally should only be doing so to check specific integrations and/or tutorials pages, since almost all other docs live in flyte.

The `ChatGPT agent` link in https://docs.flyte.org/en/latest/deployment/agents/chatgpt.html#deployment-agent-setup-chatgpt needs to be fixed in the flyte repo, which I can take care of.